doc/sphinxman/source/scf.rst

```diff
@@ -745,6 +745,24 @@ To avoid this, either set |scf__df_basis_scf| to an auxiliary
 basis set defined for all atoms in the system, or set |scf__df_scf_guess|
 to false, which disables this acceleration entirely.
 
+Linear-Scaling Exchange Build
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Methods that build exchange matrices in linear time with the size of the system (linear-time K builds) are often useful for large calculations.
+The Linear Exchange (LinK) Method, described in [Ochsenfeld:1998:1663]_, is available to use in the Direct SCF algorithm.
+LinK exploits shell pair sparsity in the density matrix, which when combined with the overlap sparsity between shell pairs, leads to linear-scaling K builds.
+LinK is most competitive when used with non-diffuse orbital basis sets, since orbital and density overlaps decay slower with diffuse functions.
+LinK is especially powerful when combined with density-matrix based ERI screening and incremental Fock builds, which decrease the number of significant two-electron integrals to calculate.
+
+NOTE: Turning on LinK is currently only recommended for research and development purposes, and not for performance,
+since a fast J matrix build compatible to use with LinK has not been developed yet.
```

Comment:
@zachglick @JonathonMisiewicz How does this sound for explaining the current state of the LinK code to users?